app/bundles/CoreBundle/Assets/css/app.css

```diff
@@ -6618,6 +6618,123 @@ a.inline-token .text-muted {
 .map-options__item.btn.active:focus {
   outline: 0 !important;
 }
+.accordion {
+  --layout-density-padding-inline-local: 16px;
+  box-sizing: border-box;
+  padding: 0;
+  border: 0;
+  margin: 0;
+  font-family: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+  inline-size: 100%;
+  list-style: none;
+}
+.accordion .panel {
+  display: list-item;
+  border: none;
+  border-radius: 0;
+  margin: 0;
+  overflow: visible;
+  -webkit-border-before: 1px solid var(--border-subtle);
+  border-block-start: 1px solid var(--border-subtle);
+  transition: border-color cubic-bezier(0.2, 0, 0.38, 0.9) 110ms;
+}
+.accordion .panel:last-child {
+  -webkit-border-after: 1px solid var(--border-subtle);
+  border-block-end: 1px solid var(--border-subtle);
+}
+.accordion .panel:not(.panel--active):last-child:hover {
+  border-block-end-color: var(--layer-hover);
+}
+.accordion *,
+.accordion *::before,
+.accordion *::after {
+  box-sizing: inherit;
+}
+.accordion-heading {
+  box-sizing: border-box;
+  font-family: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+  display: inline-block;
+  padding: 0;
+  border: 0;
+  -webkit-appearance: none;
+  appearance: none;
+  background: 0 0;
+  text-align: start;
+  position: relative;
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 0;
+  color: var(--text-primary, #161616);
+  cursor: pointer;
+  inline-size: 100%;
+  min-block-size: 40px;
+  -webkit-padding-end: var(--layout-density-padding-inline-local);
```

Comment:
I would advise just not working with padding-inline and find a solution for your styles using better supported properties such as normal padding instead of messing with browser prefixes.
I'm guessing you are using the webkit prefix, just to fix some older mobile browser? Otherwise the support seems pretty good:
- https://caniuse.com/?search=padding-inline-start
- https://caniuse.com/?search=padding-inline-end

Reply:
_I really feel bad about situations where I need to disagree but here we go:_

100% of the browsers with global usage today support padding-inline properties (checked your link)... Is there a good reason for us to keep developing Mautic for a 12-years-ago scenario?

It would only be relevant if we really want to target Internet Explorer users (which is not recommended because its security support has ended)...

Reply:
I think I did not properly convey my concern. Since you are using the webkit prefix for the inline padding stuff, I assumed you were trying to provide fallback support for some specific browsers. If that is not a concern, you should implement this padding-inline properties WITHOUT the webkit prefix.
If you DO want to support older browsers, it's better to use standard properties instead of relying on prefixes when avoidable. And this seemed like an avoidable case.
So in conclusion: feel free to use the properties if these work out better than normal padding solutions and consider dropping the webkit prefix.

Reply:
Now I got it!